bagitobjecttransfer/recordtransfer/tests/unit/test_views.py

```diff
@@ -323,12 +310,13 @@ def setUpTestData(cls):
     def setUp(self):
         _ = self.client.login(username="testuser1", password="1X<ISRUkw+tuK")
         self.session = UploadSession.new_session()
-        self.session.save()
-        self.uploaded_file = UploadedFile(
+        self.uploaded_file = TempUploadedFile(
             session=self.session,
             file_upload=SimpleUploadedFile("testfile.txt", self.one_kib),
             name="testfile.txt",
         )
+        self.session.status = UploadSession.SessionStatus.UPLOADING
+        self.session.save()
```

Comment:
Similar to my other suggestion in this file.

```suggestion
        self.session.add_temp_file(SimpleUploadedFile("testfile.txt", self.one_kib))
```

Reply:
Addressed by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/318/commits/b590646a934d658a3e69aabf9a36df474355a883